bitnami/tomcat/templates/secrets.yaml

```diff
@@ -15,3 +16,4 @@ metadata:
 type: Opaque
 data:
   tomcat-password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "tomcat-password" "providedValues" (list "tomcatPassword") "length" 10 "strong" false "context" $) }}
+{{- end }}
```

Comment:
Please ensure you include a "new line" character at the end of file as suggested by GitHub

Reply:
Done ! thanks for the feedback